src/Symfony/Component/Serializer/composer.json

```diff
@@ -18,7 +18,8 @@
     "require": {
         "php": ">=7.2.5",
         "symfony/polyfill-ctype": "~1.8",
-        "symfony/polyfill-php80": "^1.15"
+        "symfony/polyfill-php80": "^1.15",
+        "composer/semver": "^3.0"
```

Comment:
better get rid of the dependency:
- either we can use `version_compare()` instead,
- or we need to move this to require-dev, and throw some LogicException suggesting to install the dep when using the new feature

Reply:
(tests fail btw, dunno if it's related, please have a look)